include/picongpu/fields/incidentField/profiles/ExpRampWithPrepulse.def

```diff
@@ -58,6 +58,17 @@ namespace picongpu
                         static constexpr float_64 TIME_POINT_3_SI = -100.0e-15;
                         /** @} */
 
+                        static constexpr float_64 AMPLITUDE_SI = 1.0e6;
+
+                        /** Pulse duration: sigma of std. gauss for intensity (E^2)
+                     *  PREPULSE_DURATION_SI = FWHM_of_Intensity   / [ 2*sqrt{ 2* ln(2) } ]

```

Comment:
This does not fulfil our code style 
please adjust indentation to line 63 and 69. 
Otherwise, our CI gets angry